travis_install.sh

```diff
@@ -10,6 +10,9 @@ frontend() {
 
 # Install backend dependencies
 backend() {
+    npm install -g gulp-cli
+    npm install cf-icons
+    gulp copy:icons
```

Comment:
The Travis failures are weird, and I'm still trying to figure out to mitigate `coverage` finding extra `.coverage` paths that aren't actually python coverage files. 

But in the meantime, the root cause for it to be happening in this PR and not others seems to be this dependency that the backend tests now have on generated files. @chosak points out we [have logic already](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/cfgov/test.py#L22) to create placeholder files for the purpose of testing. The cleanest solution for this PR would be to remove the backend tests' dependency on cf-icons by listing all of the files that get copied in [`PLACEHOLDER_FILES`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/cfgov/test.py#L34).

Reply:
@willbarton we've reverted this.